components/src/core/components/CardTable/Decorator/DefaultCardWeb.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <oxd-card-tbody>
     <div
-      :class="classes"
       v-for="(item, index) in items"
       :key="item"
+      :class="[classes, bindClasses(index)]"
```

Comment:
Looks like we can move your `bindClasses()` to classes computed property as well. this will reduce the recurring calls to `bindClasses()` since computed property are cached.

in template
```
<div v-for="(item, index) in items" :key="item" :class="classes[index]" @click="onClick(item)($event)">
```
in computed
```
classes(): object[] {
    const highlightObject = this.tableProps.highlightRows ?? {
      rowIndexes: [],
      type: HIGHLIGHT_TYPE_SUCCESS,
    };

    return this.items.map((_, index) => {
      return {
        'oxd-table-card': true,
        [`oxd-row-highlight--${highlightObject.type}`]:
          highlightObject.rowIndexes.indexOf(index) > -1,
      };
    });
},
```

Reply:
Fixed as explained